pkg/ctl/delete/nodegroup.go

```diff
@@ -74,16 +76,32 @@ func doDeleteNodeGroup(cmd *cmdutils.Cmd, ng *api.NodeGroup, updateAuthConfigMap
 
 	stackManager := ctl.NewStackManager(cfg)
 
+	descriptions, err := stackManager.DescribeNodeGroupStacks()
```

Comment:
Any particular reason we are ignoring the error from this call?

Reply:
It's a miss on my part, I went ahead and added it in